zip-0316.rst

```diff
@@ -604,15 +612,53 @@ are reserved to indicate Metadata Items other than Receivers or
 Viewing Keys. These Items MAY affect the overall interpretation of
 the UA / UVK (for example, by specifying an expiration date).
 
+.. _`MUST-understand typecodes`:
+
+As of `Revision 1`_, the subset of Metadata typecodes in
+the range :math:`mathtt{0xE0}` to :math:`mathtt{0xEF}` are designated
+as "MUST-understand": if a Consumer is unable to recognize the meaning
```

Comment:
Consider changing "MUST-understand" to "MUST-recognize" everywhere. ("Understand" is a less accurate metaphor than "recognize".)

Reply:
Hm, I'm not sure about that. Take for example this sentence:
> As of `Revision 1`_, a Consumer of a UIVK MUST understand the meaning of all "MUST-recognise" Metadata Item typecodes present in the UIVK in order to perform Unified Address derivation, and consumers MUST follow the derivation directives specified for each such Metadata Item.

It's not just that the consumer must be able to identify the typecode, its meaning and the constraints it imposes must also be understood.